src/content/reference/react-dom/components/link.md

```diff
@@ -185,6 +185,10 @@ function SecondComponent() {
   return <link rel="stylesheet" href="second.css" precedence="low" />;
 }
 
+function ThirdComponent() {
+  return <link rel="stylesheet" href="second.css" precedence="high" />;
```

Comment:
is this supposed to be third.css? 
```suggestion
  return <link rel="stylesheet" href="third.css" precedence="high" />;
```

Reply:
Yes, thank you! Just mindless copy&paste